pkg/swagger/types.go

```diff
@@ -244,6 +244,7 @@ type Schema struct {
 	XML                  *XML                   `json:"xml,omitempty"`
 	ExternalDocs         *ExternalDocumentation `json:"externalDocs,omitempty"`
 	Example              interface{}            `json:"example,omitempty"`
+	SystemData           *Schema                `json:"systemData,omitempty"`
```

Comment:
This doesn't seem right. Why is system data in the schema? Why is it `SystemData *Schema`?

It should be a part of `OpenShiftCluster` definition. Or am I wrong?

Reply:
removed :)